src/ui/public/doc_table/components/table_row/details.html

```diff
@@ -1,6 +1,8 @@
 <td colspan="{{ columns.length + 2 }}">
-  <a class="pull-right" ng-href="#/doc/{{indexPattern.id}}/{{row._index}}/{{row._type}}/?id={{row._id | uriescape}}">
-    <small>Link to /{{row._index}}/{{row._type}}/{{row._id | uriescape}}</small></i>
-  </a>
+  <div class="pull-right">
+    <a ng-href="#/doc/{{indexPattern.id}}/{{row._index}}/{{row._type}}/?id={{row._id | uriescape}}">View</a>
+    <span style="display:inline-block; width:5px"><!--spacer--></span>
+    <a ng-click="copyTextToClipboard('#/doc/{{indexPattern.id}}/{{row._index}}/{{row._type}}/?id={{row._id | uriescape}}')">Copy</a>
```

Comment:
Thanks. I updated this to use 

```js
$detailsScope.docUrl = `#/doc/${$scope.indexPattern.id}/${$scope.row._index}/${$scope.row._type}/?id=${$scope.row._id}`;
```

But what is the equivalent of `{{row._id | uriescape}}`?